src/Microsoft.Extensions.Testing.Abstractions/project.json

```diff
@@ -1,27 +1,29 @@
 {
-    "description": "Abstractions for test runners to communicate to a tool, such as Visual Studio.",
-    "version": "1.0.0-*",
-    "repository": {
-        "type": "git",
-        "url": "git://github.com/dotnet/cli"
-    },
-    "compilationOptions": {
-        "warningsAsErrors": true,
-        "keyFile": "../../tools/Key.snk"
-    },
-    "dependencies": {
-        "Newtonsoft.Json": "7.0.1",
-        "Microsoft.DotNet.ProjectModel": "1.0.0-*",
-        "Microsoft.Extensions.Logging.Abstractions": "1.0.0-rc2-16040",
+  "description": "Abstractions for test runners to communicate to a tool, such as Visual Studio.",
+  "version": "1.0.0-*",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/dotnet/cli"
+  },
+  "compilationOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk"
+  },
+  "dependencies": {
+    "Newtonsoft.Json": "7.0.1",
+    "Microsoft.DotNet.ProjectModel": "1.0.0-*",
```

Comment:
Yes for the build it should be, I am worried about how it will be represented in the NuSpec after pack mostly.